scripts/ensure-venv.sh

```diff
@@ -27,7 +27,7 @@ if [ -n "$VIRTUAL_ENV" ]; then
 
     # TODO: Update this to strictly check .python-version
     if [ "$SENTRY_PYTHON3" = "1" ]; then
-        python -c "import sys; sys.exit(sys.version_info[:2] != (3, 6))" ||
+        python -c "import sys; sys.exit(sys.version_info[:2] != (3, 8))" ||
```

Comment:
Whoa whoah whoah, don't you think we are going too fast Jimmy? 😝

Reply:
hah. accident